docs/ref/forms/widgets.txt

```diff
@@ -496,7 +496,7 @@ These widgets make use of the HTML elements ``input`` and ``textarea``.
 
     If no ``format`` argument is provided, the default format is the first
     format found in :setting:`DATE_INPUT_FORMATS` and respects
-    :ref:`format-localization`.
+    :doc:`format-localization </topics/i18n/formatting>`.
```

Comment:
You can use
```
:doc:`/topics/i18n/formatting`
```
and this will render as it did before. As you have it now it would render as "format-localization" which isn't what we want.